WMCDuplicateRemover.Tests/EventScheduleTests.cs

```diff
@@ -1,14 +1,13 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 
 using NUnit.Framework;
 using Microsoft.MediaCenter.TV.Scheduling;
+using Moq;
 
 namespace WMCDuplicateRemover.Tests
 {
-    [Ignore]
+    [Ignore("Because they wont run? I don't remember but I'll revisit later")]
```

Comment:
I think I had started writing them and never finished so they'll have to be revisited. You're correct that the ignore attribute means that these tests will not be run. NUnit 3.x requires a comment with ignore so that's why I put it there.